client/app/components/land-use-action.js

```diff
@@ -67,31 +67,56 @@ export default class LandUseActionComponent extends Component {
     },
   ];
 
+  // Actions with partial or complete answers already saved
+  // to the PAS Form Entity in CRM
+  get existingLandUseActionSelections() {
+    const { pasForm } = this.args;
+    return this.allLandUseActionOptions.filter((option) => pasForm[option.field] || pasForm[option.attr1] || pasForm[option.attr2]);
+  }
+
+  // Actions selected by user in dropdown, or already
+  // existing in CRM
+  get landUseActionSelections() {
+    return [
+      ...this.existingLandUseActionSelections,
+      ...this.newLandUseActionSelections,
+    ];
+  }
+
+  get availableLandUseActionOptions() {
+    return this.allLandUseActionOptions.filter((option) => !this.landUseActionSelections.includes(option));
+  }
+
+  get sortedAvailableLandUseActionOptions() {
+    return this.availableLandUseActionOptions.sort((a, b) => ((a.name > b.name) ? 1 : -1));
+  }
+
   // when a user selects an option from the dropdown menu,
   // the option is removed from the landUseActionOptions
   // (and therefore removed from the dropdown).
-  // The selected option is added to the landUseActionSelections array
+  // The selected option is added to the newLandUseActionSelections array
   // (and therefore displays in the list of selected actions).
   @action
-  setSelectedActionsArray() {
-    this.landUseActionOptions.removeObject(this.selectedLandUseAction);
-    // sort options alphabetically
-    this.landUseActionOptions.sort((a, b) => ((a.name > b.name) ? 1 : -1));
+  addNewLandUseActionSelection() {
     // use unshiftObject to place new action at top of list
-    this.landUseActionSelections.unshiftObject(this.selectedLandUseAction);
+    const existsSelection = this.selectedLandUseAction.field;
```

Comment:
Hmm `existsSelection`? Why not just `selection`? We'll know it exists if it passes the check below :)

Reply:
True! 👌 Changed!